### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by azizmashkour**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Graasp-Ovarien Cycle</title>
```
Comment (juancarlosfarah) [inline public/index.html]: You can just called this `Ovarian Cycle`.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+body {
+  background: rgb(236, 236, 236);
+}
+.App {
+  text-align: center;
+}
+
```
Comment (juancarlosfarah) [inline src/App.css]: These number are a bit crazy? Can you not achieve the same thing using `flexbox`?

File: src/components/Core.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  preOvulationActive,
+  postOvulationActive,
+  handleOvulation,
+  handlePostOvulation,
+  handlePreOvulation,
+  handleStart,
+  handleStop,
+  postOvulation,
+  preOvulation,
+  secretLhFsh,
```
Comment (juancarlosfarah) [inline src/components/Core.js]: Typo `secrete` not `secret`.

File: src/components/controls/hormones/Hormones.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
+.Hormone-Container {
```
Comment (juancarlosfarah) [inline src/components/controls/hormones/Hormones.css]: @azizmashkour, you're inconsistent with the capitalisation of the classes. Sometimes it's `Upper-Upper`, sometimes it's `Upper-lower`, sometimes it's `lower-lower`. You can choose what you prefer, but be consistent!

File: src/components/controls/phases/Phases.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.Phases-Container {
+  padding: 20px;
+}
+.select-step {
+  font-size: 35px;
+}
+
+.active-ovulation, .active-preovulation, .active-postovulation {
```
Comment (juancarlosfarah) [inline src/components/controls/phases/Phases.css]: Put each class in its own line. That way it is easier to read.

```
.active-...
.active-...
.active-... {
...
```

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
+.fsh-puces, .lh-puces, .oestro-puces, .progest-puces {
+  font-size: 110px;
+  vertical-align: bottom;
+}
+
+.lh-container, .fsh-container, .oestro-container, .progest-container {
```
Comment (juancarlosfarah) [inline src/index.css]: Same as above.

File: src/index.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
+.fsh-puces, .lh-puces, .oestro-puces, .progest-puces {
```
Comment (juancarlosfarah) [inline src/index.css]: Same as above.

File: src/components/controls/phases/Phases.css (hunk 5 above)
Comment (juancarlosfarah) [reply src/components/controls/phases/Phases.css]: @azizmashkour, don't forget this?

File: src/index.css (hunk 7 above)
Comment (juancarlosfarah) [reply src/index.css]: Here too @azizmashkour.

**Comment by azizmashkour** [comment]
@juancarlosfarah Can I merge this? I already fixed all the comments and mentioned typos.

**Comment by juancarlosfarah** [comment]
Sure.

**Pull request merged**